test/util.ts

```diff
@@ -1,11 +1,16 @@
 import { readdirSync, readFileSync } from "fs";
+import * as path from "path";
+import { Block } from "../src/ast";
+import { parse } from "../src/parse";
 
 export class FileInfos {
 	readonly path: string;
-	readonly content: string;
 	constructor(readonly dir: string, readonly name: string, readonly extension: string) {
-		this.path = dir + "/" + name + extension;
-		this.content = readFileSync(this.path).toString();
+		this.path = path.join(dir, name + extension);
+	}
+
+	get content(): string {
```

Comment:
I'd prefer a method called `readContent()` for example, to explicit the fact that it is computationally expensive.

Reply:
Fixed in a226213e59d930ed7f41bad62e01bf32c017792b.